src/nunit.analyzers.tests/SameAsIncompatibleTypes/SameAsIncompatibleTypesAnalyzerTests.cs

```diff
@@ -0,0 +1,329 @@
+using Gu.Roslyn.Asserts;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Analyzers.Constants;
+using NUnit.Analyzers.SameAsIncompatibleTypes;
+using NUnit.Framework;
+
+namespace NUnit.Analyzers.Tests.SameAsIncompatibleTypes
+{
+    public class SameAsIncompatibleTypesAnalyzerTests
+    {
+        private static readonly DiagnosticAnalyzer analyzer = new SameAsIncompatibleTypesAnalyzer();
+        private static readonly ExpectedDiagnostic expectedDiagnostic =
+            ExpectedDiagnostic.Create(AnalyzerIdentifiers.SameAsIncompatibleTypes);
+
+        [Test]
+        public void AnalyzeWhenIncompatibleTypesProvided()
+        {
+            var testCode = TestUtility.WrapClassInNamespaceAndAddUsing($@"
```

Comment:
I think you can remove the $-part from all strings in this file and then also avoid the `{{` and `}}`. I can also do this and push an update to your branch, if you don't mind?

Reply:
Sure I don't mind, thanks!